host/xtest/xtest_main.c

```diff
@@ -150,6 +154,12 @@ int main(int argc, char *argv[])
 
 	opterr = 0;
 
+	if (argc > 1 && !strcmp(argv[1], "sha-perf")){
```

Comment:
BTW, what you do think if  also add checking of argv[0] ? We will able to create symlinks like ("busybox" way) 
ln -s aes-perf xtest
and just run it as $ aes-perf [params ]